src/app/core/rest/document.service.ts

```diff
@@ -49,6 +49,7 @@ export class DocumentService {
       map(response => response.headers.get('Location').split('/').pop()),
       mergeMap(id => {
         document.id = id;
+        document.creationDate = Date.now();
```

Comment:
This is not related to this PR but we should probably return document instance from back-end on creation.